x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -1,517 +1,110 @@
-= X12 EDI Connector - Mule 4
+= X12 EDI Connector 2.5.0 - Mule 4
```

Comment:
```suggestion
= X12 EDI Connector 2.5 - Mule 4
```

Reply:
Don't use x.x.x format for versions. We only update doc for minor and major releases, not patch releases.